scripts/policy/protocols/dhcp/software.zeek

```diff
@@ -45,7 +45,8 @@ event DHCP::log_dhcp(rec: DHCP::Info)
 		local id: conn_id = [$orig_h=rec$assigned_addr,
 		                     $orig_p=rec$client_port,
 		                     $resp_h=rec$server_addr,
-		                     $resp_p=rec$server_port];
+		                     $resp_p=rec$server_port,
+				     $proto=17]; # DHCP is typically UDP
```

Comment:
Did we consider defaulting the `proto` field in `conn_id` to something (e.g. whatever we agree to use for an unknown value), to avoid breaking any code that instantiates a `conn_id` record explicitly? Or did we conclude that it's sufficiently rare?